Object Analysis.ipynb

```diff
@@ -284,6 +284,22 @@
     "to_write = {}"
    ]
   },
+  {
```

Comment:
wait what is this cell? also 0.pkl -> new name

Reply:
I forgot I had deleted this cell when I was running locally bc it gave me issues with CocoDatasetNoImages, but I just added it back.